packages/live-preview-sdk/src/helpers/validation.ts

```diff
@@ -108,17 +111,17 @@ type ValidationResult = (
 export function validateLiveUpdatesConfiguration(
```

Comment:
We have the same function on live-preview-packages, should we consolidate them?

Reply:
I'm removing it from the SDK & I've updated the implementation in  [live-preview-packages](https://github.com/contentful/live-preview-packages/pull/134)